packages/fluentui/react-northstar/src/components/Datepicker/Datepicker.tsx

```diff
@@ -264,6 +277,13 @@ Datepicker.propTypes = {
   calendar: customPropTypes.itemShorthand,
   popup: customPropTypes.itemShorthand,
 
+  disabled: PropTypes.bool,
+  isRequired: PropTypes.bool,
+  onDateChange: PropTypes.func,
+  placeholder: PropTypes.string,
+  allowTextInput: PropTypes.bool,
```

Comment:
@pompomon is there a better name for this prop?

Reply:
@layershifter  I copied the name from the [FabricUI](https://github.com/microsoft/fluentui/blob/fdc095fd93f528381650e1215bcf2bfe1ff9acf6/packages/date-time/src/components/DatePicker/DatePicker.base.tsx#L477).

Reply:
ManualInput/AllowManualInput was my other option, but I think we can do that like it